azure-pipelines-cloud-db-integration.yml

```diff
@@ -71,3 +64,58 @@ stages:
           # The pip freeze output could be helpful to reproduce performance test results.
           - publish: pip-freeze.txt
             artifact: PipFreeze
+
+      - job: bigquery_expectations_test
+        timeoutInMinutes: 0 # Maximize the time that pipelines remain open (6 hours currently)
```

Comment:
Should this be 0?

Reply:
Great catch @cdkini and @donaldheppner. Currently the breakdown for each stage is the following 
- performance test < 5 min
- V3 expectations < 45 min
- V2 expectations < 1hr 30 min

A large percentage of this is I/O with BigQuery, but also a testament to the speedgains we get from going from V2 to V3. I'd like to propose that we do a 3 hr limit, which would be `180 min`

Reply:
@alexsherstinsky I think you also wanted to see this breakdown. What I also wanted to mention was that the pipeline is run 1 time per week on Sunday nights, and the entire pipeline takes < 2 hours to run.